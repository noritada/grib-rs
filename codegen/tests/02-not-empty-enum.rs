use grib_codegen::parameter_codes;

#[parameter_codes(path = "tests/data/table")]
pub enum FooCodes {
    Foo,
    Bar,
}

#[parameter_codes(path = "tests/data/table")]
pub struct BarCodes;

fn main() {}
