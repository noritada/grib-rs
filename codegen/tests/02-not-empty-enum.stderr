error: not an empty enum
 --> tests/02-not-empty-enum.rs:4:10
  |
4 | pub enum FooCodes {
  |          ^^^^^^^^

error: expected `enum`
  --> tests/02-not-empty-enum.rs:10:5
   |
10 | pub struct BarCodes;
   |     ^^^^^^
